language-server/src/main/java/org/sonarlint/languageserver/StandaloneEngineFactory.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * SonarLint Language Server
+ * Copyright (C) 2009-2018 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+package org.sonarlint.languageserver;
+
+import java.net.URL;
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+import org.sonarsource.sonarlint.core.StandaloneSonarLintEngineImpl;
+import org.sonarsource.sonarlint.core.client.api.common.LogOutput;
+import org.sonarsource.sonarlint.core.client.api.standalone.StandaloneGlobalConfiguration;
+import org.sonarsource.sonarlint.core.client.api.standalone.StandaloneSonarLintEngine;
+
+/**
+ * Create and start standalone engines, configured with loggers and extra properties.
+ */
+class StandaloneEngineFactory {
+  private final Collection<URL> analyzers;
+  private final LogOutput logOutput;
+  private final Logger logger;
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Make the "logger" logger private static final and rename it to comply with the format "LOG(?:GER)?". ([squid:S1312](https://next.sonarqube.com/sonarqube/coding_rules#rule_key=squid%3AS1312))

[See it in SonarQube](https://next.sonarqube.com/sonarqube/project/issues?id=org.sonarsource.sonarlint.core%3Asonarlint-core-parent&pullRequest=192&issues=AWND8neo6NykrOWnW1BY&open=AWND8neo6NykrOWnW1BY)

[ISSUE_KEY]: AWND8neo6NykrOWnW1BY